src/services/completions.ts

```diff
@@ -769,6 +769,7 @@ namespace ts.Completions {
             isClassLikeMemberCompletion(symbol, location)) {
             let importAdder;
             ({ insertText, isSnippet, importAdder, replacementSpan } = getEntryForMemberCompletion(host, program, options, preferences, name, symbol, location, contextToken, formatContext));
+            sortText = SortText.AutoImportSuggestions; // sortText has to be lower priority than the sortText for keywords. See #47852.

```

Comment:
Let’s make a new SortText member for this.

Reply:
do you mean a new enum member that also has value "16", or that has a new value? I think the value should be as high as possible while lower than the one for keywords, and it also has to be higher than the one for deprecated class members, but I want to check

Reply:
went ahead and created the enum member, please let me know if you meant something different.

Reply:
I meant you can bump the other numbers up so that each member gets its own value. The specific value of the number isn’t important but it seems best to avoid collisions.

Reply:
I bumped the other members down, that seemed better because otherwise the first sort text member would have to be "09", and its sort text id would be 9, and it would be wrong to just do `sortTextId.toString()` on it because we need the double digits.

Reply:
By “up” I meant a higher number so I think you did the thing I was poorly trying to describe 🙃